test/lib.bash

```diff
@@ -75,8 +75,13 @@ function serverless_operator_e2e_tests {
     kubeconfigs+=("$(pwd)/${cfg}")
   done
   kubeconfigs_str="$(array.join , "${kubeconfigs[@]}")"
+  # check for test flags
+  local run_flags="-failfast -timeout=30m -parallel=1"
+  if [ -v OPERATOR_TEST_FLAGS ] && [ ! -z "$OPERATOR_TEST_FLAGS" ]; then
```

Comment:
Nit: this line could be simplified to: `if [ -n "${OPERATOR_TEST_FLAGS:-}" ]; then`

Reply:
yes tried out this and it works fine